### ISSUE TITLE
Is this still accurate?

### ISSUE THREAD

**Issue by Caroga** [dorumd/sylius-theme-example#1]
I am trying to find a good example on how to create my own custom theme in Sylius, preferable from scratch. The documentation is very unclear on this subject, so I went on google and found this.

Is this still up to date with the latest version ?

**Comment by dorumd** [comment]
@Caroga I'm not sure, haven't testing it with sylius v1. Feel free to give it a try and if it doesn't works let me know.

**Comment by Caroga** [comment]
Sure, ill give it a try, but if I follow the documentation I think this might be outdated. 

The documentation states to overwrite the SyliusShopBundle, and it does not follow the structure as provided in this repo. 
This would matter, right? I'll be able to check later this week.

**Comment by dorumd** [comment]
@Caroga Yeah, as I remember a lot of stuff changed since I stopped working with sylius. I'll have some free time during this week and I'll update it.
